packages/pytest-simcore/src/pytest_simcore/environment_configs.py

```diff
@@ -68,3 +43,77 @@ def external_envfile_dict(request: pytest.FixtureRequest) -> EnvVarsDict:
         envs = load_dotenv(envfile)
 
     return envs
+
+
+@pytest.fixture(scope="session")
+def env_devel_dict(env_devel_file: Path) -> EnvVarsDict:
+    assert env_devel_file.exists()
+    assert env_devel_file.name == ".env-devel"
+    return load_dotenv(env_devel_file, verbose=True, interpolate=True)
+
+
+@pytest.fixture
+def mock_env_devel_environment(
+    env_devel_dict: EnvVarsDict, monkeypatch: pytest.MonkeyPatch
+) -> EnvVarsDict:
+    return setenvs_from_dict(monkeypatch, {**env_devel_dict})
+
+
+#
+# ENVIRONMENT IN A SERVICE
+#
+
+
+@pytest.fixture(scope="session")
+def service_name(project_slug_dir: Path) -> str:
+    """
+    project_slug_dir MUST be defined on root's conftest.py
```

Comment:
Would it be possible to throw this as an error somehow if `project_slug_dir` is not defined?

Reply:
pytest does that automatically.